app/src/main/java/sdp/moneyrun/menu/JoinGameImplementation.java

```diff
@@ -208,10 +208,10 @@ private void createJoinButton(Button button, int buttonId, GameRepresentation ga
 
                     double distance = gameRepresentation.getStartLocation().distanceTo(locationRep);
 
-                    if (distance > MAX_DISTANCE_TO_JOIN_GAME) {
-                        button.setEnabled(false);
-                        button.setText(activity.getString(R.string.join_game_too_far_message));
-                    }
+//                    if (distance > MAX_DISTANCE_TO_JOIN_GAME) {
```

Comment:
I think commented code should not be pushed to the master branch.

Reply:
removed.